src/components/MultiHopTrade/components/TradeSuccess/TradeSuccess.tsx

```diff
@@ -54,8 +56,12 @@ export const TradeSuccess = ({ handleBack, children }: TradeSuccessProps) => {
 
     const cryptoAmountFormatted = toCrypto(totalCryptoHumanBalance, lastHop.buyAsset.symbol)
     const chainName = adapter.getDisplayName()
-    return `You now have ${cryptoAmountFormatted} in your wallet on ${chainName}.`
-  }, [lastHop, toCrypto, totalCryptoHumanBalance])
+
+    return translate('trade.tradeComplete', {
+      cryptoAmountFormatted,
+      chainName,
+    })
+  }, [lastHop, toCrypto, totalCryptoHumanBalance, translate])
```

Comment:
leftover after attempting to render the received amount before realising that it's much hard to get that from a transaction than the updated account balance.

Leaving the translations in :)